data/json/monsters/feral_humans.json

```
@@ -74,5 +74,126 @@
     "melee_dice": 3,
     "melee_dice_sides": 8,
     "death_drops": "feral_humans_death_drops_axe"
+  },
+  {
+    "id": "mon_feral_scientist_scalpel",
+    "type": "MONSTER",
+    "name": { "str": "mad scientist" },
+    "description": "A researcher who has stared too long into the abyss and now shambles about muttering nonsense under their breath. For some reason, the zombies don't seem to mind the fact that this person is obviously still alive. They're clutching a tiny razor-sharp blade in one shaky hand.",
+    "default_faction": "zombie",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 3,
+    "melee_cut": 2,
+    "dodge": 1,
+    "harvest": "CBM_SCI_FERAL",
+    "vision_day": 30,
+    "vision_night": 3,
+    "death_drops": "feral_scientists_death_drops_scalpel",
+    "death_function": [ "NORMAL" ],
+    "zombify_into": "mon_zombie_scientist",
+    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "HUMAN",
+      "CAN_OPEN_DOORS",
+      "PATH_AVOID_DANGER_2",
+      "DROPS_AMMO",
+      "CBM_SCI"
+    ]
+  },
+  {
+    "id": "mon_feral_labsecurity_9mm",
+    "type": "MONSTER",
+    "name": { "str": "feral security guard" },
+    "description": "This security guard still breathes, but has been lost to some terrible madness. They move among the undead, handgun at the ready and saucer pupils scanning for threats in a mockery of their former duty.",
+    "default_faction": "zombie",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 2,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "armor_bash": 4,
+    "armor_cut": 6,
+    "armor_bullet": 4,
+    "armor_stab": 4,
+    "dodge": 1,
+    "harvest": "human",
+    "vision_day": 30,
+    "vision_night": 3,
```

```suggestion
    "vision_night": 4,
    "path_settings": { "max_dist": 10 },
```
Vision night boosted a bit by higher assumed perception. They have even more reason than the feral scientists to know the area, although their zombies have lost it.